### PR TITLE
Additive code changes for file move

### PR DESCRIPTION
**Pull request by aaronFranssell**
As we start scripting out/sharing code between these ingestion Lambdas we can start by reviewing the Lambda code itself.

### CODE REVIEW

File: data_processor.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import boto3
+import urllib
+import os
+import time
+import json
+from datetime import datetime
+
+CHUNK_SIZE = 1073741824 # 1 GB
+
+logger = logging.getLogger()
```
Comment (aaronFranssell) [inline data_processor.py:21]: I don't know of any advantages or disadvantages of printing vs logging...

Comment (TiannaW24) [reply data_processor.py:21]: The logger utility I've seen is just slightly formatted printing....

File: data_processor.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    logging.info(f'source_key: {source_key}')
+
+    file_size = event['Records'][0]['s3']['object']['size']
+
+    target_bucket = os.environ['TARGET_DATA_BUCKET']
+    target_data_folder_path = bucket_destination_mapping()[source_bucket]
+    now = datetime.now()
+    target_key = os.path.join(target_data_folder_path, now.strftime('%Y'), now.strftime('%m'), now.strftime('%d'), source_key)
+    if 'atri' in target_key:
+        1/0
```
Comment (dweibler) [inline data_processor.py]: Why " if 'atri' in target_key: 1/0 "

File: data_processor.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    raw_source_key = event['Records'][0]['s3']['object']['key']
+    logging.info(f'raw_source_key: {raw_source_key}')
+    source_key = urllib.parse.unquote_plus(event['Records'][0]['s3']['object']['key'])
+    logging.info(f'source_bucket: {source_bucket}')
+    logging.info(f'source_key: {source_key}')
+
+    file_size = event['Records'][0]['s3']['object']['size']
+
+    target_bucket = os.environ['TARGET_DATA_BUCKET']
+    target_data_folder_path = bucket_destination_mapping()[source_bucket]
```
Comment (dweibler) [inline data_processor.py:35]: ?? Would it make sense to put the raw-submissions bucket within the BUCKET_PATH_MAPPING dictionary as well as the folder_path?  I know right now we have a single raw-submissions bucket, but if down we need to split the data-lake up into one bucket per data provider for example....  Just a thought

**Review by dweibler — APPROVED** [review]

**Review by TiannaW24 — APPROVED** [review]

File: data_processor.py (hunk 2 above)
Comment (aaronFranssell) [reply data_processor.py]: That was in the branch for testing cloudwatch alarms... Refresh? That shouldn't be there anymore.

File: data_processor.py (hunk 3 above)
Comment (aaronFranssell) [reply data_processor.py:35]: Yes thats an excellent point, one I'm hoping to resolve with Dan in the meeting I have in one minute.

If we do end up splitting out buckets then I can look into streamlining this.

File: data_processor.py (hunk 1 above)
Comment (JAMSUPREME) [reply data_processor.py:21]: I'd prefer to do it a consistent way, but otherwise no strong opinion

**Review by JAMSUPREME — APPROVED** [review]

**Pull request merged**